test/clib/test_ctreactor.cpp

```diff
@@ -13,6 +13,15 @@ TEST(ctreactor, reactor_soln)
     int sol = soln_newSolution("gri30.yaml", "gri30", "none");
     int reactor = reactor_new3("IdealGasReactor", sol, "test");
     ASSERT_EQ(reactor, 0);
+
+    int ret = reactor_setName(reactor, "spam");
+    ASSERT_EQ(ret, 0);
+    int buflen = reactor_name(reactor, 0, 0);
+    char* buf = new char[buflen];
```

Comment:
Instead of `new` and `delete`, you can use `vector<char> buf(buflen)`. I think there are one or two other `clib` tests where that's what we use.

Reply:
There are quite a few instances.